src/Components/AddTask.jsx

```diff
@@ -0,0 +1,36 @@
+import React , { useState} from "react";
+import Button from "./Button";
+
+function AddTask({ addTask }) {
+    const [taskInput, setTaskInput] = useState("")
+    const [status, setStatus] = useState("Todo")
```

Comment:
In this file, some lines has semicolon at the end and some don't. Why like that?